mmv1/products/tags/api.yaml

```diff
@@ -0,0 +1,95 @@
+# Copyright 2021 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Tags
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://cloudresourcemanager.googleapis.com/v3/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: 'name'
+    base_url: '{{op_id}}'
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: 'response'
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: 'done'
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: 'error'
+    message: 'message'
+objects:
+  - !ruby/object:Api::Resource
+    name: 'TagKey'
+    min_version: beta
+    base_url: tagKeys
+    self_link: "tagKeys/{{name}}"
+    update_verb: :PATCH
+    update_mask: true
+    description: A TagKey, used to group a set of TagValues.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/resource-manager/docs/tags/tags-creating-and-managing'
+      api: 'https://cloud.google.com/resource-manager/reference/rest/v3/tagKeys'
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        description: |
+          The generated numeric id for the TagKey.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: parent
+        description: |
+          Input only. The resource name of the new TagKey's parent. Must be of the form organizations/{org_id}.
```

Comment:
Can this be a project instead of an organization?

Reply:
I imagine that's the plan, but for now the documentation specifically says parent has to be formatted as an organization. Do you think I should check whether using a project works? https://cloud.google.com/resource-manager/reference/rest/v3/tagKeys